js/views/playingScreen.js

```diff
@@ -79,11 +79,12 @@ class PlayingScreen extends React.Component {
    */
   handleVrAnimationEnd(ref, stateName) {
     if (ref) {
+      const _this = this;
```

Comment:
Would it be better to declare `animationEndHandler` as a method of this component? Ideally we would also remove the listener on `componentWillUnmount` so using a method would allow us to do so and also allow us to avoid the `_this = this` declaration.

Reply:
JPFranco AFAIK, moving animationEndHandler into class level method will not help to fix execution context `this`, due to it's being called in event handling flow, when context is `window`. Shortest way to achieve fixed context is to set in 82 line something like: `const animationEndHandler = () => {...};`

Reply:
\+

Reply:
@zelibobla yes, of course. What I had in mind was to then either bind the handler in the constructor or use a class property (although that would require an additional babel plugin). The main point here is that it's probably a good idea to remove the listener when the component un-mounts. 

Anyway, I think its unlikely that the component will actually unmount with an active listener, but I think it's a good practice to remove DOM listeners on mount. I think this is fine though